contentcuration/contentcuration/utils/publish.py

```diff
@@ -708,18 +728,40 @@ def process_image_strings(content, zf, channel_id):
                     logging.warning("NOTE: the following error would have been swallowed silently in production")
                     raise
 
-            image_name = "images/{}.{}".format(checksum, ext[1:])
-            if image_name not in zf.namelist():
-                with storage.open(ccmodels.generate_object_storage_name(checksum, filename), 'rb') as imgfile:
```

Comment:
Prior to the changes here, it looks like reading from storage only occurred within the above `if` condition, which seems to test whether the file exists in the perseus zip.